robocop/checkers/spacing.py

```diff
@@ -54,14 +54,14 @@ def parse_file(self):
                 if empty_lines > 1:
                     self.report("too-many-trailing-blank-lines", lineno=len(self.raw_lines), col=0)
                     return
-            if not empty_lines and not last_line.endswith('\n'):
+            if not empty_lines and not last_line.endswith(('\n','\r')):
                 self.report("missing-trailing-blank-line", lineno=len(self.raw_lines), col=0)
 
     def check_line(self, line, lineno):
         self.raw_lines.append(line)
 
-        stripped_line = line.rstrip('\n')
-        if stripped_line and stripped_line[-1] == ' ':
+        stripped_line = line.rstrip('\n\r')
```

Comment:
And here it's inverted

Reply:
rstips removes all characters that are given in the parameter, regardless of the order